src/test/examples.ml

```diff
@@ -154,9 +164,7 @@ let downloader () =
         if_seq (filename_ov =$= no_value)
           ~t:begin
             let filename =
-              url >> exec ["sed"; "s:.*/\\([^\\?\\/]*\\)\\?.*:\\1:"]
-              |> output_as_string
-            in
+              no_newline_sed ~input:url "s/.*\\/\\([^?\\/]*\\).*/\\1/" in
```

Comment:
If possible can you use a different separator (ie, `:` instead of `/`), as in the previous `sed` call? Though if this is all going away at some point, not a big deal.